### PR TITLE
Od 99 - Clarify the difference between Upload and URL Link

### PR DESCRIPTION
**Pull request by sivang**

### CODE REVIEW

File: ckanext/ed/schemas/dataset.json [hunk 1]
```diff
@@ -127,7 +127,9 @@
       "field_name": "url",
       "label": "URL",
       "required": true,
-      "preset": "resource_url_upload"
+      "preset": "resource_url_upload",
+      "help_text": "CLARIFYING TEXT FOR UPLOAD"
```
Comment (zelima) [inline ckanext/ed/schemas/dataset.json]: This is not really helpful, think we need an actual text here. Let's come up with one, smth like "Please upload a new file or import using sourve link" (we can brainstorm on this)

Comment (zelima) [reply ckanext/ed/schemas/dataset.json]: Also, have you actually seen it? it looks quite ugly to me cc @Tanvir - it's looking more like "Name"s (the field below) description than link's. Can we do anything about it?

image

File: ckanext/ed/schemas/dataset.json [hunk 2]
```diff
@@ -127,7 +127,9 @@
...
+
```
Comment (zelima) [inline ckanext/ed/schemas/dataset.json]: Let's get rid of unnecessary blank lines

File: ckanext/ed/schemas/dataset.json (hunk 1 above)
Comment (sivang) [reply ckanext/ed/schemas/dataset.json]: I propose "You can upload a new file or import data using the source link"

Regarding the "ugliness" this is known and approved by @JGulic , where as @tanvir will adjust it and fix the appearance.

File: ckanext/ed/schemas/dataset.json (hunk 2 above)
Comment (sivang) [reply ckanext/ed/schemas/dataset.json]: done

File: ckanext/ed/schemas/dataset.json (hunk 1 above)
Comment (zelima) [reply ckanext/ed/schemas/dataset.json]: Like your version of help text :+1:

**Comment by zelima** [comment]
Think this is fine to merge

**Pull request merged**